File: Data Structures & Algorithms/counting-bits/submission-0.java
class Solution {
    public int[] countBits(int n) {
        
        int res[]= new int[n+1];
        res[0]=0;
        int x ;
        int count=0;
        for(int i = n ;i>0;i--)
        {
            x=i;
            while(x!=0)
            {
                x = x&(x-1);
                count++;
            }
            res[i]=count;
            count=0;


        }
    
        return res;
    }
    
}

File: Data Structures & Algorithms/daily-temperatures/submission-0.py
class Solution:
    def dailyTemperatures(self, temperatures: List[int]) -> List[int]:
        n=len(temperatures)
        res = [0]*n
        for i in range(n):
            curr=temperatures[i]
            count=0
            for j in range(i+1,n):
                if temperatures[j]>curr:
                    res[i]=j-i
                    break
        return res
        



        